base/file.jl

```diff
@@ -224,6 +221,13 @@ julia> readdir()
 julia> readdir("test")
 1-element Array{String,1}:
  "dir"
+
+julia> mkpath("test/actually_a_directory.txt") # creates two directories
+"test/actually_a_directory.txt"
+
+julia> isdir("test/actually_a_directory.txt")
```

Comment:
better actually to rename this, since I think we actually already have the `test` directory from the previous bit of the example (so "creates two directories" is not right-- we only make one).
```suggestion
julia> mkpath("intermediate_dir/actually_a_directory.txt") # creates two directories
"intermediate_dir/actually_a_directory.txt"

julia> isdir("intermediate_dir/actually_a_directory.txt")
```